package-lock.json

```diff
@@ -935,9 +1173,9 @@
             }
         },
         "node_modules/@types/express-serve-static-core": {
-            "version": "4.17.35",
-            "resolved": "https://registry.npmjs.org/@types/express-serve-static-core/-/express-serve-static-core-4.17.35.tgz",
-            "integrity": "sha512-wALWQwrgiB2AWTT91CB62b6Yt0sNHpznUXeZEcnPU3DRdlDIz74x8Qg1UUYKSVFi+va5vKOLYRBI1bRKiLLKIg==",
+            "version": "4.17.36",
+            "resolved": "https://registry.npmjs.org/@types/express-serve-static-core/-/express-serve-static-core-4.17.36.tgz",
+            "integrity": "sha512-zbivROJ0ZqLAtMzgzIUC4oNqDG9iF0lSsAqpOD9kbs5xcIM3dTiyuHvBc7R8MtWBp3AAWGaovJa+wzWPjLYW7Q==",
```

Comment:
Sadly, this is still unwanted because is totally unrelated to opc-ua. Some thing for eslint above. We could merge the PR anyway but I would be happier if I see only changes connected with the upgrade of the opc-ua packages.